tests/static-api/_expected/v1/repos.json

```diff
@@ -43,11 +47,15 @@
       "branch_protections": [
         {
           "pattern": "master",
-          "ci_checks": [
```

Comment:
This change breaks anything (namely sync-team) that relies on the current schema?

Don't think we can do this (at least not immediately).

Reply:
I don't think that there's anything else than `sync-team` that read the branch protections, and the `sync-team` change is [ready](https://github.com/rust-lang/sync-team/pull/76). I already did some breaking changes recently that broke sync-team, it doesn't seem like a big deal.

Although, it would be bad if this broke other tools that just reuse the schema, even if they don't read the branch protections at all. Not sure if there are any of these though. Is there even any other tool apart from `sync-team` that reads the `repos` endpoint?